components/admin_console/workspace-optimization/dashboard.tsx

```diff
@@ -244,26 +246,70 @@ const WorkspaceOptimizationDashboard = (props: Props) => {
     }, []);
 
     const accData: AccordionItemType[] = Object.entries(data).map(([accordionKey, accordionData]) => {
-        const items = accordionData.items.map((item) => (
-            <AccordionItem
-                key={`${accordionKey}-item_${item.id}`}
-                iconColor={getItemColor(item.status)}
-            >
-                <h5><i className={classNames('icon', {'icon-check-circle-outline': item.status === 'none', 'icon-alert-outline': item.status === 'warning', 'icon-alert-circle-outline': item.status === 'error'})}/>{item.title}</h5>
-                <p>{item.description}</p>
-            </AccordionItem>
-        ));
+        const chipsInfo: { [key in ChipsInfoKey]: number } = {
+            info: 0,
+            warning: 0,
+            error: 0,
+            none: 0,
+        };
+        const items: React.ReactNode[] = [];
+        accordionData.items.forEach((item) => {
+            items.push((
+                <AccordionItem
+                    key={`${accordionKey}-item_${item.id}`}
+                    iconColor={getItemColor(item.status)}
+                >
+                    <h5><i className={classNames('icon', {'icon-check-circle-outline': item.status === 'none', 'icon-alert-outline': item.status === 'warning', 'icon-alert-circle-outline': item.status === 'error'})}/>{item.title}</h5>
+                    <p>{item.description}</p>
+                </AccordionItem>
+            ));
+            if (chipsInfo[item.status] !== undefined) {
+                chipsInfo[item.status] += 1;
+            }
```

Comment:
1. `chipsInfo[item.status] !== undefined` is always true
2. regarding the comment below we should still add a guard here: `if (item.status) {`

Reply:
right, it was needed before to prevent items with no status.